test/controllers/sold.land.filter.controller.spec.ts

```diff
@@ -91,6 +97,36 @@ describe("SOLD LAND FILTER controller", () => {
       expect(resp.status).toEqual(500);
       expect(resp.text).toContain(SERVICE_UNAVAILABLE);
     });
+
+    test(`renders the ${config.SOLD_LAND_FILTER_PAGE} page with updated back link when 
+      save and resume feature flag is active`, async () => {
+      mockIsActiveFeature.mockReturnValueOnce(false); // SHOW_SERVICE_OFFLINE_PAGE
```

Comment:
Try to mock [serviceAvailabilityMiddleware](https://github.com/companieshouse/overseas-entities-web/blob/9bab1a6c7fe9b2ca1cfec15e5d8fc76767ebc3f8/test/controllers/check.your.answers.controller.spec.ts#L117) as Mark suggested, it will simplify the flag mocking to just one